driver/rds_utils.cc

```diff
@@ -133,6 +133,24 @@ std::string RDS_UTILS::get_rds_cluster_host_url(std::string host) {
   return f(AURORA_CHINA_CLUSTER_PATTERN);
 }
 
+std::string RDS_UTILS::get_rds_cluster_id(std::string host) {
+  auto f = [host](const std::regex pattern) {
+    std::smatch m;
+    if (std::regex_search(host, m, pattern) && m.size() > 1) {
+      return m.size() > 1 ? m.str(1) : std::string("");
+    }
```

Comment:
Don't really need the check for `m.size() > 1`, if the pattern matches it'll be there